src/agentor/core/agent.py

```
@@ -508,6 +509,51 @@ async def aresume(self, run_id: str):
         """Async variant of resume()."""
         return await self._loop.aresume(run_id)
 
+    def fork(
+        self,
+        run_id: str,
+        input: Optional[MessageInput] = None,
```

Ruff (0.16.2)

[error] Function argument `input` is shadowing a Python builtin

(A002)